### PR TITLE
Support monitoring PostgreSQL slow SQLs.

### PR DESCRIPTION
**Pull request by yswdqz**
- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [x] Update the documentation to include this new feature.
- [x] Tests(including UT, IT, E2E) are added to verify the new feature.
- [x] If it's UI related, attach the screenshots below.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

}~T@YXFC~K6@HRCZ`LTSNCL
W)0OGX(T1S%~E0PMMA@)W~O

### CODE REVIEW

File: docs/en/setup/backend/backend-postgresql-monitoring.md [hunk 1]
```diff
@@ -44,8 +45,33 @@ PostgreSQL monitoring provides monitoring of the status and resources of the Pos
...
 You can customize your own metrics/expression/dashboard panel.
 The metrics definition and expression rules are found in `/config/otel-rules/postgresql.yaml`.
 The PostgreSQL dashboard panel configurations are found in `/config/ui-initialized-templates/postgresql`.
+
+## Collect sampled slow SQLs
+SkyWalking leverages [fluentbit](https://fluentbit.io/) or other log agents for collecting slow SQL statements from PostgreSQL.
+
+### Data flow
+1. fluentbit agent collects slow sql logs from PostgreSQL.
+2. fluentbit agent sends data to SkyWalking OAP Server using native meter APIs via HTTP.
```
Comment (kezhenxu94) [inline docs/en/setup/backend/backend-postgresql-monitoring.md:58]:
```suggestion
2. fluentbit agent sends data to SkyWalking OAP Server using native log APIs via HTTP.
```

File: docs/en/setup/backend/backend-postgresql-monitoring.md [hunk 2]
```diff
@@ -44,8 +45,33 @@ PostgreSQL monitoring provides monitoring of the status and resources of the Pos
...
+3. The SkyWalking OAP Server parses the expression with [LAL](../../concepts-and-designs/lal.md) to parse/extract and store the results.
+
+### Set up
+1. Set up [fluentbit](https://docs.fluentbit.io/manual/installation/docker).
+2. Config [fluentbit](../../../../test/e2e-v2/cases/postgresql/postgres-exporter/fluent-bit.conf)
+3. Config PostgreSQL to enable slow log.[example](../../../../test/e2e-v2/cases/postgresql/postgres-exporter/postgresql.conf).
```
Comment (kezhenxu94) [inline docs/en/setup/backend/backend-postgresql-monitoring.md:64]:
```suggestion
3. Config PostgreSQL to enable slow log. [Example](../../../../test/e2e-v2/cases/postgresql/postgres-exporter/postgresql.conf).
```

File: test/e2e-v2/cases/postgresql/postgres-exporter/docker-compose.yml [hunk 3]
```diff
@@ -20,23 +20,42 @@ services:
     extends:
       file: ../../../script/docker-compose/base-compose.yml
       service: oap
+    entrypoint: ['sh', '-c', '/download-mysql.sh && chmod 777 /skywalking/docker-entrypoint.sh && /skywalking/docker-entrypoint.sh']
```
Comment (kezhenxu94) [inline test/e2e-v2/cases/postgresql/postgres-exporter/docker-compose.yml:23]: This is only needed when we use mysql driver, but psql does not use mysql driver, right?

```suggestion
```

Comment (yswdqz) [reply test/e2e-v2/cases/postgresql/postgres-exporter/docker-compose.yml:23]: Yes, I forget to remove it.

**Review by wankai123 — APPROVED** [review]
LGTM

**Review by kezhenxu94 — DISMISSED** [review]

File: docs/en/changes/changes.md [hunk 4]
```diff
@@ -19,6 +19,7 @@
 * Remove physical index existing check and keep template existing check only to avoid meaningless `retry wait`
   in `no-init` mode.
 * Make sure instance list ordered in TTL processor to avoid TTL timer never runs.
+* Support monitoring PostgreSQL slow SQLs.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:22]: You had a `* Update MySQL dashboard to visualize collected slow SQLs.` change log, I think you miss the one for PostgreSQL?

Comment (yswdqz) [reply docs/en/changes/changes.md:22]: yes

**Review by wu-sheng — APPROVED** [review]
LGTM, waiting for e2e testing.

**Comment by wu-sheng** [comment]
Let's bring slow SQL collecting for PostgreSQL and MySQL server to the showcase.

**Pull request merged**